src/www/ui/api/index.php

```diff
@@ -160,6 +160,8 @@ function (\Slim\Routing\RouteCollectorProxy $app) {
     $app->post('', GroupController::class . ':createGroup');
     $app->delete('/{id:\\d+}', GroupController::class . ':deleteGroup');
     $app->delete('/{id:\\d+}/user/{uid:\\d+}', GroupController::class . ':deleteGroupMember');
+    $app->put('/{id:\\d+}/user/{userId:\\d+}', GroupController::class . ':changeUserPermission');
```

Comment:
The function `changeUserPermission` does not exists in this PR. Please remove it.

Reply:
Reminder @dushimsam